dashboard.csss-ci.com/server/api/models/Services_model.php

```diff
@@ -3,46 +3,78 @@
     exit('No direct script access allowed');
 
 class Services_model extends CI_Model {
-
+    /**
+     * table/database/query for services
+    */
     protected $table = "services";
-
+    /**
+     * inserts data within the query
+     * @param Array $data - data to be inserted
+     * @return boolean - true if sucessful
+    */
     public function save($data) {
         return $this->db->insert($this->table, $data);
     }
-
+    /**
+     * finds the row and data of desired location
+     * @param Array/String $where - location 
+     * @return Array - desired row and data from query
+    */
     public function find($where) {
         return $this->db->select('*')
                         ->from($this->table)
                         ->where($where)
                         ->get()
                         ->result();
     }
-
+    /**
+     * get all rows from query
+     * @param null
+     * @return Array - with the number of rows from query
```

Comment:
getAll returns an array with all of the rows of the table, not the number of rows from the query